src/main/java/frc/robot/commands/ForceOuttakeCommand.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.OuttakeSubsystem;
+
+public class ForceOuttakeCommand extends CommandBase {
```

Comment:
This command should be renamed to ForceOuttakeSubsystemModeCommand for same reasons as below. Use f2 for mass rename!